code/datums/perks/fate.dm

```diff
@@ -0,0 +1,197 @@
+	..()
+	holder.stats.addTempStat(STAT_ROB, 10, INFINITY, "Fate Alcoholic")
+
+/datum/perk/alcoholic_active/remove()
+	holder.stats.addTempStat(STAT_ROB, "Fate Alcoholic")
+	..()
+
+/datum/perk/noble
+	name = "Noble"
+	icon_state = "family" //https://game-icons.net
+	desc = "You're a noble, a graceful ornament to the civil order. The jewel of society. Waiting to be stolen."
+
+/datum/perk/noble/assign(mob/living/carbon/human/H)
+	..()
+	if(!holder.last_name)
+		qdel(src)
+	holder.sanity.environment_cap -= 1
+	var/turf/T = get_turf(holder)
+	var/obj/item/W = pickweight(list(/obj/item/weapon/tool/knife/butterfly = 1,
+				/obj/item/weapon/tool/knife/switchblade = 1,
+				/obj/item/weapon/tool/knife = 1,
+				/obj/item/weapon/tool/knife/boot = 0.5,
+				/obj/item/weapon/tool/knife/hook = 2,
+				/obj/item/weapon/tool/knife/ritual = 0.5,
+				/obj/item/weapon/tool/scythe = 0.3,
+				/obj/item/weapon/tool/sword = 0.2,
+				/obj/item/weapon/tool/sword/katana = 0.2,
+				/obj/item/weapon/tool/knife/butch = 2,
+				/obj/item/weapon/tool/knife/dagger = 0.8))
+	W = new W(T)
+	W.name = "[holder.last_name] family [W.name]"
+	holder.put_in_hands(W)
+
+/datum/perk/noble/remove()
+	holder.sanity.environment_cap += 1
+	..()
+
+/datum/perk/rat
+	name = "Rat"
+	desc = "You have been always clever, but it is one thing to be clever and another to be wise."
+	icon_state = "rat"
+
+/datum/perk/rat/assign(mob/living/carbon/human/H)
+	..()
+	holder.sanity.max_level -= 10
+
+/datum/perk/rat/remove()
+	holder.sanity.max_level += 10
+	..()
+
+/datum/perk/rejected_genius
+	name = "Rejected genius"
+	icon_state = "knowledge"
+	desc = "Your dreams are undisturbed by reality and you are in search of the philosopher’s stone and the elixir of life."
+
+/datum/perk/rejected_genius/assign(mob/living/carbon/human/H)
+	..()
+	holder.sanity.environment_cap -= 1
+	holder.sanity.positive_prob_multiplier -= 1
+	holder.sanity.insight_passive_gain_multiplier *= 1.5
+	holder.sanity.max_level -= 20
+
+/datum/perk/rejected_genius/remove()
+	holder.sanity.environment_cap += 1
+	holder.sanity.positive_prob_multiplier += 1
+	holder.sanity.insight_passive_gain_multiplier /= 1.5
+	holder.sanity.max_level += 20
+	..()
+
+/datum/perk/oborin_syndrome
+	name = "Oborin Syndrome" //https://game-icons.net/1x1/delapouite/prism.html
+	icon_state = "prism"
+	desc = "You see world in negative, as if it was an old noir movie."
+
+/datum/perk/oborin_syndrome/assign(mob/living/carbon/human/H)
+	..()
+	holder.sanity.max_level += 20
+	holder.species.taste_sensitivity = TASTE_NUMB
+	spawn(1)
+		holder.update_client_colour() //Handle the activation of the colourblindness on the mob.
+
+/datum/perk/oborin_syndrome/remove()
+	holder.sanity.max_level -= 20
+	..()
+
+/datum/perk/lowborn
+	name = "Lowborn"
+	icon_state = "ladder" //https://game-icons.net/1x1/delapouite/hole-ladder.html
+	desc = "For them you are a paria, thrash — untouchable! That’s the word! You are an Untouchable!"
```

Comment:
Is thrash the right word here? Is it supposed to be trash?

Reply:
"Paria" is spelt "pariah".